lib/Room.php

```diff
@@ -726,48 +726,14 @@ public function setLastActivity(\DateTime $now): bool {
 
 	/**
 	 * @param \DateTime $since
-	 * @param int $callFlag
-	 * @param bool $isGuest
-	 * @return bool
+	 * @return void
 	 */
-	public function setActiveSince(\DateTime $since, int $callFlag, bool $isGuest): bool {
-		if ($isGuest && $this->getType() === self::TYPE_PUBLIC) {
-			$update = $this->db->getQueryBuilder();
-			$update->update('talk_rooms')
-				->set('active_guests', $update->createFunction($update->getColumnName('active_guests') . ' + 1'))
-				->set(
-					'call_flag',
-					$update->expr()->bitwiseOr('call_flag', $callFlag)
-				)
-				->where($update->expr()->eq('id', $update->createNamedParameter($this->getId(), IQueryBuilder::PARAM_INT)));
-			$update->executeStatement();
-
+	public function setActiveSince(\DateTime $since, int $callFlag, bool $isGuest): void {
+		$this->activeSince = $since;
+		$this->callFlag = $callFlag;
```

Comment:
The database does:
```php
$update->expr()->bitwiseOr('call_flag', $callFlag)
```

So you also need to bitwise combine it here:

```suggestion
		$this->callFlag = $this->callFlag | $callFlag;
```